src/components/Trade/TradeConfirm/AssetToAsset.tsx

```diff
@@ -40,15 +40,15 @@ export const AssetToAsset = ({
 
   const renderIcon = () => {
     return status === chainAdapters.TxStatus.Confirmed ? (
-      <Circle bg={green} w='100%' h='100%'>
+      <Circle bg={green} size='100%'>
```

Comment:
OK, double-checked the doc and this abstraction exists for `<Center />` components.